pkg/infoschema/interface.go

```diff
@@ -30,7 +30,14 @@ type InfoSchema interface {
 	context.MetaOnlyInfoSchema
 	TableByName(ctx stdctx.Context, schema, table ast.CIStr) (table.Table, error)
 	TableByID(ctx stdctx.Context, id int64) (table.Table, bool)
-	SchemaNameByTableID(tableID int64) (ast.CIStr, bool)
+	// TableItemByID returns a lightweight table meta specified by the given ID,
+	// without loading the whole info from storage.
```

Comment:
Please put it in a more straightforward and less ambiguous way, like "It's all in-memory operation", "It doesn't involve network or IO operations".